sbt-dotty/sbt-test/source-dependencies/macro-expansion-dependencies-2/test

```diff
@@ -1,15 +1,23 @@
+$ copy-file changes/MacroRuntimeCompileError.scala MacroRuntime.scala
+> clean
+> compile
+
+$ copy-file changes/MacroRuntimeRuntimeError.scala MacroRuntime.scala
+> clean
+> compile
```

Comment:
It seems to simply repeat the same steps as in the first section.

Reply:
The first checks that `MacroRuntimeCompileError` compiles and the second checks that `MacroRuntimeRuntimeError` compiles. Notice the `RuntimeCompile`/`RuntimeRuntime` in the file names.